ansible/controllerProvision.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+- hosts: localhost
+  connection: local
+  tasks:
+  - name: Add controller to inventory
+    add_host:
+      name: masterController
+      ansible_host: "{{ controllerIP }}"
+      ansible_user: root
+      ansible_ssh_pass: contrail123
+      ansible_ssh_extra_args: '-o UserKnownHostsFile=/dev/null'
+      groups: controller
+
+- hosts: masterController
+  roles:
+    - controller
```

Comment:
We think that it will be better to add:

```
- hosts: controller:
  roles:
  - controller
```

at the end of `vmware-deploy-testenv.yml`. Correct IP address is already obtained in `vmware-vm` role: https://github.com/codilime/contrail-windows-ci/blob/development/ansible/roles/vmware-vm/tasks/post_deploy.yml#L21. Filter must be added there to accept only addresses from correct subnets (i.e. `10.84.*` and `172.17.*`)

Reply:
(that way, we won't need to add the controller to the inventory – it would already be there)

Reply:
> https://github.com/codilime/contrail-windows-ci/blob/development/ansible/roles/vmware-vm/tasks/post_deploy.yml#L21. Filter must be added there to accept only addresses from correct subnets (i.e. `10.84.*` and `172.17.*`)

Don't we need to add this filter there? Or is `first` sufficient?